modules/microshift-install-multus-running-cluster.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-cni-multus.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-multus-installing-on-running-cluster_{context}"]
+= Installing the Multus CNI plugin on an running cluster
+
+If you want to attach additional networks to a pod for high-performance network configurations, you can install the {microshift-short} Multus RPM package. After installation, a host restart is required to recreate all the pods with the Multus annotation.
+
+[IMPORTANT]
+====
+The Multus RPM should not be removed after you install it.
+====
+
+.Procedure
+
+. Install the Multus RPM package by running the following command:
++
+[source,terminal]
+----
+$ sudo dnf install microshift-multus
+----
++
+[TIP]
+====
+If you create your custom resources (CRs) before completing your installation of Multus, you can avoid restarting the service a second time to apply them.
+====
+
+. To apply the manifest from the package to an active cluster, restart the host by running the following command:
++
+[source,terminal]
+----
+$ sudo systemctl restart
+----
+
+.Verification
+
+. Ensure that the Multus CNI plugin components are created after restarting by running the following command:
```

Comment:
```suggestion
. After restarting, ensure that the Multus CNI plugin components are created by running the following command:
```
Just a thought to avoid all the gerunds at the end of the sentence.